api/database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UsersTableSeeder::class);
+        $this->call(MaterialsTableSeeder::class);
```

Comment:
This seeder can stay now for presentational purposes but should be removed afterwards.